src/networkcloud/azext_networkcloud/aaz/operations/custom_properties.py

```diff
@@ -0,0 +1,198 @@
+
+    @classmethod
+    def _build_schema_error_detail_read(cls, _schema):
+        if cls._schema_error_detail_read is not None:
+            _schema.additional_info = cls._schema_error_detail_read.additional_info
+            _schema.code = cls._schema_error_detail_read.code
+            _schema.details = cls._schema_error_detail_read.details
+            _schema.message = cls._schema_error_detail_read.message
+            _schema.target = cls._schema_error_detail_read.target
+            return
+
+        cls._schema_error_detail_read = _schema_error_detail_read = AAZObjectType()
+
+        error_detail_read = _schema_error_detail_read
+        error_detail_read.additional_info = AAZListType(
+            serialized_name="additionalInfo",
+            flags={"read_only": True},
+        )
+        error_detail_read.code = AAZStrType(
+            flags={"read_only": True},
+        )
+        error_detail_read.details = AAZListType(
+            flags={"read_only": True},
+        )
+        error_detail_read.message = AAZStrType(
+            flags={"read_only": True},
+        )
+        error_detail_read.target = AAZStrType(
+            flags={"read_only": True},
+        )
+
+        additional_info = _schema_error_detail_read.additional_info
+        additional_info.Element = AAZObjectType()
+
+        _element = _schema_error_detail_read.additional_info.Element
+        _element.type = AAZStrType(
+            flags={"read_only": True},
+        )
+
+        details = _schema_error_detail_read.details
+        details.Element = AAZObjectType()
+        cls._build_schema_error_detail_read(details.Element)
+
+        _schema.additional_info = cls._schema_error_detail_read.additional_info
+        _schema.code = cls._schema_error_detail_read.code
+        _schema.details = cls._schema_error_detail_read.details
+        _schema.message = cls._schema_error_detail_read.message
+        _schema.target = cls._schema_error_detail_read.target
+
+    @classmethod
+    def _build_schema_properties_read(cls, _schema):
+        if cls._schema_properties_read is not None:
+            _schema.exitCode = cls._schema_properties_read.exitCode
+            _schema.outputHead = cls._schema_properties_read.outputHead
+            _schema.resultUrl = cls._schema_properties_read.resultUrl
+            return
+
+        cls._schema_properties_read = _schema_properties_read = AAZObjectType()
+
+        properties_read = _schema_properties_read
+        properties_read.exitCode = AAZStrType(
+            flags={"read_only": True},
+        )
+        properties_read.outputHead = AAZStrType(
+            flags={"read_only": True},
+        )
+        properties_read.resultUrl = AAZStrType(
+            flags={"read_only": True},
+        )
+
+        _schema.exitCode = cls._schema_properties_read.exitCode
+        _schema.outputHead = cls._schema_properties_read.outputHead
+        _schema.resultUrl = cls._schema_properties_read.resultUrl
```

Comment:
Why this is not defined in swagger directly?

Reply:
We are RPaaS Service and async operation is defined and provided to us by RPaaS. We have no control over the swagger definition for the operationStatuses API. However we are allowed to add extra properties in API response, and the CLI custom code handles the properties. 

Reference Link RPaaS: https://armwiki.azurewebsites.net/rpaas/async.html

The majority of our resources are served by K8s bridge acting as User-RP.